ext/reflection/tests/ReflectionFunction_getParameter_error1.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+ReflectionFunction::getParameter()
+--FILE--
+<?php
+$function = new ReflectionFunction('sort');
+var_dump($function->getParameter('Array'));
+?>
+--EXPECTF--
+Fatal error: Uncaught ReflectionException: The parameter specified by its name could not be found in %s:3
+Stack trace:
+#0 %s: ReflectionFunctionAbstract->getParameter('Array')
+#1 {main}
+  thrown in %s on line 3
```

Comment:
Please merge all the error cases together and use a try/catch block to catch and print the exceptions.